specs/sifive.com/PRCI/RESET/0.1.0/RESET_rtl.json5

```diff
@@ -0,0 +1,42 @@
+{
+  abstractionDefinition: {
+    vendor: 'sifive.com',
+    library: 'PRCI',
+    name: 'RESET_rtl',
+    version: '0.1.0',
+    busType: {
+      vendor: 'sifive.com',
+      library: 'PRCI',
+      name: 'RESET',
+      version: '0.1.0',
+    },
+    ports: {
+      RESET: {
+        description: 'reset signals',
+        wire: {
+          isReset: true,
+          onMaster: {direction: 'out', presence: 'required'},
+          onSlave:  {direction: 'in', presence: 'required'}
+        }
+      }
+    },
+    props: {
+      ResetPolarity: {
+          type: "string",
+	  enum: ["ActiveHigh", "ActiveLow"],
+      },
+      ResetType: {
+	  type: "string",
+	  enum: ["Synchronous", "Asynchronous", "Full Asynchronous"]
+      },
+      ClocksBeforeDeassertion: {
+	  type: "integer",
+	  description: "Number of clock edges that must be provided before this reset signal is deasserted in order to properly initialize and clear contention within the IP"
+      },
+      ClocksAfterDeassertion: {
+	  type: "integer",
```

Comment:
should we assume `minimum: 0` here?

Reply:
i think 0 for Asynchronous and 1 for synchronous is reasonable assumption. 0 for synchronous doesnt make any sense

Reply:
I think what Megan says is true for the before case.  I think 0 for afterdeassertion for both cases is ok.  Afterdeassertion is intended to indicated after the IP is let out of reset but before it can be used.

Reply:
I added min of 1, along with a comment in the description that we'd ideally like to do a conditional minimum